styles/core/forms.less

```diff
@@ -13,25 +13,21 @@
 * not, you can obtain one from Tidepool Project at tidepool.org.
 */
 
-// Form controls
-// ====================================
+@import 'variables.less';
 
-@form-control-border-size:   2px;
-@form-control-height:        (30px + 2*@form-control-border-size);
-@form-control-border-color:  @purple-light;
 
 .form-control-border {
```

Comment:
OK yeah, please add a brief section to the doc re: naming conventions for the CSS modules stuff vs. the old stuff.